src/components/component-playground.jsx

```diff
@@ -270,6 +270,10 @@ module.exports = React.createClass({
     window.addEventListener('resize', this.onWindowResize);
 
     this._updateContentFrameOrientation();
+
+    if(this.props.component) {
+      this.refs["componentName-" + this.props.component].getDOMNode().scrollIntoView(); 
```

Comment:
scrollIntoView() without any parameters is supported in all major browsers.

![screen shot 2016-07-15 at 1 40 26 pm](https://cloud.githubusercontent.com/assets/19645753/16871819/bc3829c8-4a91-11e6-8183-35a7caae04ba.png)